### PR TITLE
Add functions to aladin backend to return functions that convert coor…

### PR DESCRIPTION
**Pull request by cquiroz**
…ds to pixels and viceversa

### CODE REVIEW

File: demo/src/main/scala/demo/AladinContainer.scala [hunk 1]
```diff
@@ -191,7 +193,11 @@ object AladinContainer {
           r.backend.addCatalog(cat) *>
           updateSvgState.flatMap { s =>
             r.backend.recalculateView *> r.backend.runOnAladinCB(updateVisualization(s))
-          }
+          } *>
+          r.backend.world2pixFn
+            .flatMap(f => Callback.log(s"${f(testCoord).get._1}x${f(testCoord).get._2}")) *>
+          r.backend.pix2worldFn
+            .flatMap(f => Callback.log(s"${f(48742, 49134).get}"))
```
Comment (toddburnside) [inline demo/src/main/scala/demo/AladinContainer.scala:200]: Did you want to keep this here?

**Review by toddburnside — APPROVED** [review]
LGTM

File: demo/src/main/scala/demo/AladinContainer.scala (hunk 1 above)
Comment (cquiroz) [reply demo/src/main/scala/demo/AladinContainer.scala:200]: Yeah. this is for the demo, essentially this proves me that the functions are doing the right thing

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**